packages/scope-manager/src/ScopeManager.ts

```diff
@@ -40,25 +41,15 @@ class ScopeManager {
   /**
    * The root scope
    */
+  readonly #options: ScopeManagerOptions;
   public globalScope: GlobalScope | null;
```

Comment:
The JSDoc comment didn't move

Reply:
Fixed by https://github.com/azat-io/eslint-plugin-perfectionist/issues/97 🚀